modules/core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/blob/BlobManager.java

```diff
@@ -95,22 +95,28 @@ default BlobProvider getBlobProviderWithNamespace(String id) {
     BlobProvider getBlobProvider(Blob blob);
 
     /**
-     * INTERNAL - Gets an {@link InputStream} for the data of a managed blob. Used by internal implementations, regular
-     * callers should call {@link Blob#getStream}.
+     * Gets an {@link InputStream} for the data of a managed blob.
+     * <p>
+     * If the blob is managed this is equivalent to {@link ManagedBlob#getStream()}, otherwise returns {@code null}.
      *
      * @param blob the blob
-     * @return the stream
+     * @return the stream, or {@code null} if the blob is not managed
+     * @deprecated since 11.1
```

Comment:
The Javadoc says `If the blob is managed this is equivalent to {@link ManagedBlob#getStream()}, otherwise returns {@code null}` so I think it's enough?

Reply:
But I'll add something anyway ok.